types/zwjsbridge/index.d.ts

```diff
@@ -0,0 +1,504 @@
+// Type definitions for non-npm package ZWJSBridge API - zwjsbridge.js 1.0
+// Project: https://assets.zjzwfw.gov.cn/assets/ZWJSBridge/1.0.1/zwjsbridge.js
+// Definitions by: Yuxiang Ren <https://github.com/shlyren>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.2
+
+interface ZWJSBridge {
+    /**
+     * 初始化jsapi，初始化完成即onReady之后再调用jsapi。
+     * @param callBack 初始化成功回调
+     */
+    onReady(callBack: () => void): void
+    /////// 缓存 //////
+    /**
+     * 存储数据缓存
+     */
+    setLocalStorage(options: {
+        key: string
+        value: string
+	}): Promise<{
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/zwjsbridge/index.d.ts;
  Line: 20;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.